packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5757,6 +5777,10 @@ abstract class ComponentElement extends Element {
           ],
         ),
       );
+      // _Make sure the old child subtree are deactivated and disposed.
+      try {
```

Comment:
This isn't enough. If you inflate a widget during the process and a descendant Element fails to activate, the current pattern in framework.dart is to replace the entire subtree rooted at the `performRebuild` call with an ErrorWidget. So some inflated element are still active (since the inflation process was partially successful).

Reply:
Is this only a best effort thing? should we have a catch or finally to clean up the reset of tree?